Blocks/HTTP/impl/posix_server.h

```diff
@@ -281,10 +283,10 @@ class HTTPServerPOSIX final {
 
             // `route_for_directory` must have leading slash and must not have trailing slash except if it's a root.
             const std::string route_for_directory =
-                options.route_prefix +
-                ((options.route_prefix == "/" || path_components_empty) ? "" : "/") +
+                options.route_prefix + ((options.route_prefix == "/" || path_components_empty) ? "" : "/") +
                 path_components_joined;
```

Comment:
@dkorolev What I don't like in this coding style is that the shape of the code changes based on the line length, and if there are multiple pieces of code with the same shape, they could be transformed to different shapes by the indent.

### Before
```
const std::string route_for_directory =
    options.route_prefix +
    ((options.route_prefix == "/" || path_components_empty) ? "" : "/") +
    path_components_joined;
```
```
const std::string route_for_file =
    route_for_directory +
    (route_for_directory == "/" ? "" : "/") +
    item_info.basename;
```
```
std::string trailing_slash_redirect_url =
    options.public_url_prefix +
    (options.public_url_prefix.back() == '/' ? "" : "/") +
    (path_components_empty ? "" : path_components_joined + "/");
```

-------
### After
```
const std::string route_for_directory =
    options.route_prefix + ((options.route_prefix == "/" || path_components_empty) ? "" : "/") +
    path_components_joined;
```
```
const std::string route_for_file =
    route_for_directory + (route_for_directory == "/" ? "" : "/") + item_info.basename;
```
```
std::string trailing_slash_redirect_url = options.public_url_prefix +
                                          (options.public_url_prefix.back() == '/' ? "" : "/") +
                                          (path_components_empty ? "" : path_components_joined + "/");
```